docs-en/chapter_preface/about_the_book.md

```diff
@@ -1,45 +1,45 @@
 # The Book
 
-The aim of this project is to create an open source, free, novice-friendly introductory tutorial on data structures and algorithms.
+The aim of this project is to create an open source, free, beginner-friendly crash course on data structures and algorithms.
 
-- Animated graphs are used throughout the book to structure the knowledge of data structures and algorithms in a way that is clear and easy to understand with a smooth learning curve.
-- The source code of the algorithms can be run with a single click, supporting Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig and other languages.
-- Readers are encouraged to help each other and make progress in the chapter discussion forums, and questions and comments can usually be answered within two days.
+- Animated illustrations, structured delivery for data structures and algorithms, with easy-to-understand content, and a smooth learning curve.
+- Run code with just one click, supporting Java, C++, Python, Go, JS, TS, C#, Swift, Rust, Dart, Zig and other languages.
+- Readers are encouraged to help each other in the comment and discussion section for each chapter, questions and comments are usually answered within two days.
 
 ## Target Readers
 
-If you are a beginner to algorithms, have never touched an algorithm before, or already have some experience brushing up on data structures and algorithms, and have a vague understanding of data structures and algorithms, repeatedly jumping sideways between what you can and can't do, then this book is just for you!
+If you are a beginner to algorithms, you have never learned algorithms before, or you have accumulated some experience in algorithms, but you only have a vague understanding of data structures and algorithms, and you are constantly jumping between "Yep" and "hmm", then this book is for you!
```

Comment:
"Yep" and "hmm"
|
"yep" and "hmm"

Should we preserve their consistency?

Reply:
Sorry this is a typo, will fix it.